app/models/container_group.rb

```diff
@@ -94,4 +94,18 @@ def disconnect_inv
     self.deleted_on = Time.now.utc
     save
   end
+
+  def self.post_refresh_ems(ems_id, _ems_refresh_start_time)
+    self.reconnect_events(ems_id)
+  end
+
+  def self.reconnect_events(ems_id)
+    events = EmsEvent.where(:ems_id => ems_id, :container_group_id => nil).where.not(:container_group_name => nil)
```

Comment:
The query makes sense to me, can you add to be on the safe side `:source => 'KUBERNETES'`.
we can have `:source => 'DATAWAREHOUSE'` with container_group_name not empty